configure.ac

```diff
@@ -205,16 +205,16 @@ PHP_PROG_AWK
 PHP_PROG_BISON([3.0.0])
 PHP_PROG_RE2C([0.13.4])
 
+RE2C_FLAGS="-i --no-generation-date"
```

Comment:
I'm in favor of using `--no-debug-info` instead of `-i`, is more readable :)

Reply:
I guess we can add that also, yes. Added.